es-ds-docs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nuxt-app",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "nuxt build",
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare"
+  },
+  "dependencies": {
+    "@energysage/es-bs-base": "^2.0.4",
+    "@nuxtjs/google-fonts": "^3.2.0",
+    "nuxt": "^3.12.4",
+    "vue": "latest"
```

Comment:
Should we restrict this to 3.X?

Reply:
Weird that the standard Nuxt setup did this - I've set it to the latest specific version of Vue, we can upgrade manually as we need to.